.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Pull request
+about: Create a pull request to contribute new features or bug fixes
+title: "[PULL REQUEST]"
+labels: pull_request
+
+---
+
+## Describe your changes
+
+## Related issue number (if any)
+
+## Checklist before requesting a review
+- [ ] I have performed a self-review of my code.
+- [ ] If it is a core feature, I have added thorough tests.
+- [ ] All tests are passing.
```

Comment:
Isn't this checked by our GH actions?

Reply:
I'm editing to say "Integration tests are passing"